src/Factory.php

```diff
@@ -102,6 +102,16 @@ final public static function createSequence(iterable|callable $sequence): array
         return static::new()->sequence($sequence)->create();
     }
 
+    /**
+     * @param list<mixed> $values
+     *
+     * @return list<T>
+     */
+    final public static function createForEach(string $field, array $values): array
```

Comment:
Do you think this is common enough to warrant a dedicated static method?

Reply:
nope you're right, I'll remove it